31Aug_MajorityElement.py

```diff
@@ -0,0 +1,33 @@
+# https://leetcode.com/problems/majority-element/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+#####################################################################################################################################################
+#
+# Given an array nums of size n, return the majority element.
+# The majority element is the element that appears more than ⌊n / 2⌋ times. You may assume that the majority element always exists in the array.
+#
+# Example 1:
+# Input: nums = [3,2,3]
+# Output: 3
+#
+# Example 2:
+# Input: nums = [2,2,1,1,1,2,2]
+# Output: 2
+#
+class Solution:
+    def majorityElement(self, nums: List[int]) -> int:
+      # nums = [3, 2, 3]
+      a = Counter(nums)
+      # a = Counter({3: 2, 2: 1})
+      max_value = 0
+      result = 0
+      for i, j in a.items():
+        # i = 3, j = 2
+        # i = 2, j = 1
+        if j > max_value:
+          result = i
+          max_value = j
+          # result = 3
+          # max_value = 2
+      return result
```

Comment:
If you add this to the bottom of the code it'll run in a python environment
```
if __name__ == "__main__":
    assert Solution().majorityElement([2,2,1,1,1,2,2]) == 2
    assert Solution().majorityElement([3,2,3]) == 3
```

Reply:
We do not have main here in Leetcode. All have pre-defined function, on top of it we code the solution. And we have enough test cases/3 during Run and even more test cases are executed, when we submit.

Reply:
So what are you doing? Copy pasting the code back and forth between Leetcode and Github?